Sources/WebDriver/Session.swift

```diff
@@ -73,6 +73,14 @@ public class Session {
             Requests.SessionTimeouts(session: id, type: type, ms: duration * 1000))
     }
 
+    public func synchronous(javascript script: String,  args: [String]) throws {
```

Comment:
We should name this `execute` to match the protocol spec, and default to following the `script` parameter name from the spec as well.